api/kyverno/v1/rule_types.go

```diff
@@ -32,7 +32,7 @@ type Rule struct {
 	// criteria can include resource information (e.g. kind, name, namespace, labels)
 	// and admission review request information like the name or role.
 	// +optional
-	ExcludeResources ExcludeResources `json:"exclude,omitempty" yaml:"exclude,omitempty"`
+	ExcludeResources MatchResources `json:"exclude,omitempty" yaml:"exclude,omitempty"`
```

Comment:
Should we rename `MatchResource` to some generic name ?

Reply:
`MatchResource` sounds good to me, do you have a better name to suggest ?